app/models/podcast_episode.rb

```diff
@@ -3,6 +3,9 @@ class PodcastEpisode < ApplicationRecord
     duration_in_seconds
   ]
 
+  include PgSearch::Model
+  multisearchable against: %i[title body_text] # spike, limited attributes
```

Comment:
It's a total coincidence that I'm searching against the same attributes here, I'll add some distinct ones in the future.